src/helpers/tests/flow-chart.testData.ts

```diff
@@ -223,7 +223,7 @@ export const flowArguments: FlowArguments = {
       type: "string",
     },
     {
-      name: "hide_exc_info",
```

Comment:
why did you change the name here?

Reply:
Two reasons: 1) according to the meaning of the option's description "If set, then exception stack information to be added to the logging message, useful in debugging", which implies the name should be `show_exc_info` instead of `hide_exc_info`; 2) based on the UI of Pod Edit Modal, it shows the option as "show_exc_info". 

If the option suppose to be `hide_exc_info`, the description and the text on Pod Edit Modal should be corrected. I wonder which one the option does indicate, and whether the default behaviour is showing or hiding the exc info. 

Thanks.

Reply:
One quick thing, this is test data so these exact flow arguments won't be the ones displaying in the dashboard. We fetch flow arguments from jinaAPI or jinad based on connection status, we don't have control over these arguments or their naming in the dashboard. We should leave this unchanged as it is a copy of what we pulled from jina api.

Reply:
Hi @imsergiy , thanks for your comment. Currently, I suppose there's a mocked server which located at `dev-resources` directory, and all the flow arguments are pulled from there?  The mocked argument name for the input is `show_exc_info`, I guess that's why at the dashboard Pod Edit Modal it shows `show_exc_info`.

The argement's name at mocked server is different from the one at the `flow-chart.testData.ts` file, which caused a bit trouble, because at the cypress `podView.ts` test, the argument's name is used for targeting the input element. If the name is set as `hide_exc_info`, Cypress won't find it, due to the current argument's name from mocked server is 'show_exc_info'.

If the mocked server's response is the same with the ones at `flow-chart.testData.ts`, would it be a bit easier to understand and test? To keep the argument identical, I can either change the argument name at `flow-chart.testData.ts`  and mocked server to `hide_exc_info`, or change them to `show_exc_info`?

What do you guys think? Thanks. @BastinJafari @imsergiy

Reply:
I see, good point. Either way works then since it's just mock data.

Reply:
It would be good to keep it consistent between testServer and dashboard, I like your idea.

Reply:
Hi @imsergiy @BastinJafari , to keep the data consistent, the argument's name `show_exc_info` is changed to `hide_exc_info`. However, according to the argument's description "If set, then exception stack information to be added to the logging message, useful in debugging", I'm a bit confused about the expected default behaviour, whether the info will be logged or not. Anyway, I suppose it can be fixed at another PR.